_posts/2017-06-21-exceptional_servant_handling.markdown

```diff
@@ -138,19 +138,20 @@ convert :: IO a -> Handler a
 convert = Handler . ExceptT . try
 ```
 
+**NOTE**: This `convert` function is a no-op. `try`'s `e` type is instantiated to `ServerError` here, and no `IO a` actions you pass will throw `ServerError`, meanwhile any `IOException` exception thrown by IO actions will not be caught by `try`. So `convert` is basically doing nothing.
```

Comment:
The point of this blog post is using `IO` exceptions for the `ServantErr`, which the above `server` does.

```suggestion
**NOTE**: This `convert` function *only handles `ServantErr`. `try`'s `e` type is instantiated to `ServantErr` here. Any `IOException` exception thrown by IO actions will not be caught by `try` here.
```